### PR TITLE
tests(django) Pin to Django < 4.0.0

### PR DESCRIPTION
**Pull request by brettlangdon**
## Commit Message
{{title}}

Only pin for now to unblock CI, then we can address the test failures.

```

self = <django.core.cache.CacheHandler object at 0x7f10307f4e20>, alias = 'redis'

    def create_connection(self, alias):
        params = self.settings[alias].copy()
        backend = params.pop('BACKEND')
        location = params.pop('LOCATION', '')
        try:
            backend_cls = import_string(backend)
        except ImportError as e:
>           raise InvalidCacheBackendError(
                "Could not find backend '%s': %s" % (backend, e)
            ) from e
E           django.core.cache.backends.base.InvalidCacheBackendError: Could not find backend 'django_redis.cache.RedisCache': cannot import name 'smart_text' from 'django.utils.encoding' (/root/project/.riot/venv_py3100_django~400/lib/python3.10/site-packages/django/utils/encoding.py)

.riot/venv_py3100_django~400/lib/python3.10/site-packages/django/core/cache/__init__.py:41: InvalidCacheBackendError
```

### CODE REVIEW

**Comment by brettlangdon** [comment]
@Mergifyio backport 0.56

File: ddtrace/contrib/django/patch.py [hunk 1]
```diff
@@ -484,7 +484,8 @@ def _patch(django):
     # DEV: this check will be replaced with import hooks in the future
     if "django.conf.urls.static" not in sys.modules:
         import django.conf.urls.static
-    trace_utils.wrap(django, "conf.urls.url", traced_urls_path(django))
+    if django.VERSION < (4, 0, 0):
```
Comment (Kyle-Verhoog) [inline ddtrace/contrib/django/patch.py:487]: Can we revert these changes since they're not related to the actual disabling/pinning of tests?

Comment (brettlangdon) [reply ddtrace/contrib/django/patch.py:487]: oh sure, I was originally thinking "why not", will help keep the user's app from crashing, but since we cannot verify that we won't cause a runtime issue (rather than startup like this one), probably better to have a startup app crash

**Review by Kyle-Verhoog — APPROVED** [review]
